mmv1/products/compute/BackendService.yaml

```diff
@@ -1304,4 +1304,5 @@ properties:
           the field must be in [0, 1]. This configures the sampling rate of requests to the load balancer
           where 1.0 means all logged requests are reported and 0.0 means no logged requests are reported.
           The default value is 1.0.
+        default_from_api: true
```

Comment:
Can we set this to be `Default: 1.0` instead? That's what the documentation has, and I'd prefer that than it become Optional and Computed.